client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -92,3 +106,17 @@ function PaymentInfoBlockWrapper( {
 		</aside>
 	);
 }
+
+function BackupPaymentMethodNotice() {
+	const translate = useTranslate();
+	/* translators: Notice that this purchase may use a backup payment method if it fails to renew */
+	const noticeText = translate( 'May use backup' );
+	return (
+		<a
+			className="manage-purchase__backup-payment-method-notice"
+			href="https://wordpress.com/support/payment/#manage-payment-methods"
```

Comment:
Given https://github.com/Automattic/wp-calypso/pull/56665, would it be better to link to the user's Payment Methods page (e.g. https://wordpress.com/me/purchases/payment-methods or the site-specific version) here?  That way they can actually change the backup status right away.  And from that page, the above pull request already links to these support docs too.

Reply:
🤔  I like the thought, but I'm not convinced that's better. The documentation link explains directly what it is and how to change it, whereas if we linked to the list of payment methods, it doesn't really explain anything and there could be tons of unrelated payment methods listed. It'd be nice if we could link to a payment method detail page, but we don't have such a thing.

Reply:
To me it seemed pretty unexpected to link to a documentation page for the purpose of sending the user back to a different Calypso page afterwards -- it's a little bit ping-pong-y :smile:  (Also the link is not styled to look like an external/help link, which made it unexpected also.)

I think the most likely thing the user is going to want to do if they click this is to see what their "backup payment methods" currently are -- and https://wordpress.com/me/purchases/payment-methods is the direct place to do that.  Once they get there, the checkboxes next to every row in https://github.com/Automattic/wp-calypso/pull/56665 should make that pretty clear already and not necessarily require the user to read the documentation (but if they want to, the "Learn more" links there really seem like a cleaner way to send them to it).

Separately, the draft documentation page doesn't seem to link the user directly to their payment methods page, although that part can be fixed (I will comment at 535-gh-Automattic/payments-shilling).

I wonder what others think?

Reply:
Alright, that's fine. We can always change the target later.